src/jep/Jep.java

```diff
@@ -129,7 +132,7 @@ public void run() {
                      * while another thread is in python, then the thread state
                      * can get messed up leading to stability/GIL issues.
                      */
```

Comment:
Oh, that's a good point.

Thanks for bearing with my bad threading code. It's been a couple years since I've done any non-trivial threading and that was all in C# and never with native modules.